server/src/main/java/com/cloud/network/vpc/VpcManagerImpl.java

```diff
@@ -266,6 +273,12 @@ public class VpcManagerImpl extends ManagerBase implements VpcManager, VpcProvis
     @Qualifier("networkHelper")
     protected NetworkHelper networkHelper;
     @Inject
+    private NsxService nsxService;
+    @Inject
```

Comment:
```suggestion
```

Reply:
This fixed the simulator CI.
NsxService cannot be used in normal build without '-Dnoredist'.